src/lib/common/sem.cpp

```diff
@@ -451,12 +461,14 @@ int release_curl_context(struct curl_context *pcc, bool final)
 
     if (s != 0)
     {
-      LM_E(("pthread_mutex_unlock"));
+      LM_E(("KZ: pthread_mutex_unlock"));
       return s;
     }
 
     pcc->pmutex = NULL; // It will remain in global map
   }
+  else
+    LM_M(("KZ: NOT unlocked pmutex"));
```

Comment:
I'm running a valgrind test right now. If it turns out OK, these LM_Ms are not needed and will be removed